### ISSUE TITLE
Paru error while loading shared libraries/Firefox homepage not working

### ISSUE THREAD

**Issue by TrippieMint** [gh0stzk/dotfiles#318]
While trying to update some packages I got this error: paru: error while loading shared libraries: libalpm.so.14: cannot open shared object file: No such file or directory

![Shot-2024-09-15-164101](https://github.com/user-attachments/assets/cc6e1afa-2aca-4673-b9d3-aec663de2c4d)

not only that but the firefox homepage seems to be a bit broken 
![image](https://github.com/user-attachments/assets/464b05b9-3c05-4bbb-bea6-ac076756f318)

**Comment by TrippieMint** [comment]
for the first issue I think spoofing my mac address caused the problem

**Comment by gh0stzk** [comment]
No no, is not about spoofing your mac or my dotfiles.

Pacman released yesterday a major version, so that breaks paru, yay, pamac or any other aur helper. 

in the case of the paru, reading issues in the paru repo, recompile it doesnt work.

Just this morning I updated and as expected paru broke, a solution that they give and which I followed is to install paru-git from aur, that's what I did and it works.

```
git clone https://aur.archlinux.org/paru-git
cd paru-git
makepkg -si
```

this will fix paru, and libalpm issues.

about your firefox and connection problem, i dont know bro, for sure is not related to my dots.
